django/contrib/postgres/fields/hstore.py

```diff
@@ -51,6 +51,20 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(HStoreField, self).formfield(**defaults)
 
+    def get_prep_value(self, value):
+        value = super(HStoreField, self).get_prep_value(value)
+
+        if isinstance(value, dict):
+            prep_value = {}
+            for key, val in value.items():
+                if not isinstance(key, six.string_types):
+                    key = str(key)
```

Comment:
Could you add some tests with non-ASCII values? I guess we probably want to use `force_text()` here instead of casting to bytes on Python 2.